app/helpers/storage_service_helper/textual_summary.rb

```diff
@@ -15,7 +15,7 @@ def textual_group_properties
 
   def textual_group_capabilities
     capabilities = @record.capabilities.map do |c|
-      [c["name"], c["value"]]
+      [c.first, c.last]
```

Comment:
Why not use the same style as above?  https://github.com/ManageIQ/manageiq-ui-classic/pull/8656/files#diff-5d79f85ea7f83c5d5e76301aa67d72c60ed594f1d1e0fd47a3c54b93ecf6a573R51
`do |key, value|` is much preferred over `c.first, c.last` since it is more obvious what is happening to the reader

Reply:
I will try that now and test it

Reply:
@agrare fixed as your request.